### PR TITLE
Correctly sort History columns

### PR DESCRIPTION
**Pull request by autogun**
Sorting history columns introduced in https://github.com/bugy/script-server/pull/430 does not take into account that by default, `sort()` method sorts elements alphabetically. This leads to an unwanted sort result of: `[1, 10, 100, 2, 3]`

This PR enhances sorting to sort numerically, by date property etc.

### CODE REVIEW

File: samples/scripts/test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (bugy) [inline samples/scripts/test.sh:1]: Is this file needed?

**Review by bugy — CHANGES_REQUESTED** [review]
One small comment, otherwise looks good

File: samples/scripts/test.sh (hunk 1 above)
Comment (autogun) [reply samples/scripts/test.sh]: Definitely not, slipped in by mistake :-)

**Review by bugy — APPROVED** [review]
Thanks!

**Comment by bugy** [comment]
Hi, i found another issue: script name sorting considers letter case, could
you fix it also, please? :)

Danny Rehelis ***@***.***> schrieb am Mi., 9. Juni 2021,
12:46:

**Review by bugy — APPROVED** [review]

**Comment by bugy** [comment]
Thanks again!
